doc/migrating-to-1-3.md

```diff
@@ -0,0 +1,106 @@
+# Migrating to 1.3
+
+The 1.3 plugin introduces several breaking changes to better deal with multiple GraphQL endpoints (services), lazy configuration and package name configuration. These changes only affect the plugin. The generated code is unchanged and your consumer code doesn't require any change but your `build.gradle` (or `build.gradle.kts`) files might.
+
+### Plugin id is now `com.apollographql.apollo`
+
+We renamed the plugin id from `com.apollographql.android` to `com.apollographql.apollo` to make it clear that the plugin works also for non Android projects. `com.apollographql.android` will be removed in a future revision.
+
+```groovy
+// Don't do this
+apply plugin: 'com.apollographql.android'
+
+// Do this
+apply plugin: 'com.apollographql.apollo'
+```
+### Specifying schema and graphql files location
```

Comment:
I think it would be nice to keep the order according to order of which these take affect. Mentioning first about service usage and multiple service support can come first. Then this section with even more custom configuration may come afterwards. What do you think?

Reply:
Or maybe having a link to other plugin configuration doc can be an option too.

Reply:
Yes, for both.